ansible/install_dependencies.sh

```diff
@@ -18,9 +18,15 @@ sudo apt-get update
 
 # Install python dependencies
 echo Installing Python dependencies
-sudo apt-get install -y python3-distutils
-sudo apt-get install -y python3-pip
-python3 -m pip install -U pip setuptools
+sudo apt-get install -y python3-distutils python3-testresources python3-pip
+python3 -m pip install -U pip
+
+# Update setuptool version if it is higher than 65
```

Comment:
nit: A detailed comment as to why this is being done will help any one looking at the code.
# Downgrade the setuuptools version to 65 if it is higher to prevent breaking changes (or whatever the reason is)

Reply:
Done